<?xml version="1.0" encoding="utf-8"?>
<tests>
  <compliant>
    <public>-----BEGIN PUBLIC KEY-----\nQyNTUxOQAAACCtrF27B/zd9DEpd38IbVBy93wSeYXKU0AGXMyO8ePu2QAAAKBSzpYEUs6W\n-----END PUBLIC KEY-----</public>
  </compliant>
  <noncompliant>
    <access>-----BEGIN PRIVATE KEY-----\nQyNTUxOQAAACCtrF27B/zd9DEpd38IbVBy93wSeYXKU0AGXMyO8ePu2QAAAKBSzpYEUs6W\n-----END PRIVATE KEY-----</access>
    <key>
      -----BEGIN RSA KEY-----
      QyNTUxOQAAACCtrF27B/zd9DEpd38IbVBy93wSeYXKU0AGXMyO8ePu2QAAAKBSzpYEUs6W
      -----END RSA KEY-----
    </key>
    <rsa>-----BEGIN RSA PRIVATE KEY-----\nQyNTUxOQAAACCtrF27B/zd9DEpd38IbVBy93wSeYXKU0AGXMyO8ePu2QAAAKBSzpYEUs6W\n-----END RSA PRIVATE KEY-----</rsa>
    <dsa>-----BEGIN DSA PRIVATE KEY-----\nQyNTUxOQAAACCtrF27B/zd9DEpd38IbVBy93wSeYXKU0AGXMyO8ePu2QAAAKBSzpYEUs6W\n-----END DSA PRIVATE KEY-----</dsa>
    <ec>-----BEGIN EC PRIVATE KEY-----\nQyNTUxOQAAACCtrF27B/zd9DEpd38IbVBy93wSeYXKU0AGXMyO8ePu2QAAAKBSzpYEUs6W\n-----END EC PRIVATE KEY-----</ec>
    <openssh>-----BEGIN OPENSSH PRIVATE KEY-----\nQyNTUxOQAAACCtrF27B/zd9DEpd38IbVBy93wSeYXKU0AGXMyO8ePu2QAAAKBSzpYEUs6W\n-----END OPENSSH PRIVATE KEY-----</openssh>
  </noncompliant>
</tests>